pkg/jobs/schedule_metrics.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package jobs
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/util/metric"
+)
+
+// ExecutorMetrics describes metrics related to scheduled
+// job executor operations.
+type ExecutorMetrics struct {
+	NumStarted   *metric.Counter
+	NumSucceeded *metric.Counter
+	NumFailed    *metric.Counter
+}
+
+var _ metric.Struct = &ExecutorMetrics{}
+
+// MetricStruct implements metric.Struct interface
+func (m *ExecutorMetrics) MetricStruct() {}
+
+// SchedulerMetrics are metrics specific to job scheduler daemon.
+type SchedulerMetrics struct {
+	// Number of schedules that were ready to execute.
+	ReadyToRun *metric.Gauge
+	// Number of scheduled jobs started.
+	NumStarted *metric.Gauge
+	// Number of jobs started by schedules that are currently running.
+	NumRunning *metric.Gauge
+	// Number of schedules rescheduled due to SKIP policy.
+	RescheduleSkip *metric.Gauge
+	// Number of schedules rescheduled due to WAIT policy.
+	RescheduleWait *metric.Gauge
+	// Number of schedules that could not be processed due to an error.
+	NumBadSchedules *metric.Counter
+}
+
+// MakeSchedulerMetrics returns metrics for scheduled job daemon.
+func MakeSchedulerMetrics() SchedulerMetrics {
+	return SchedulerMetrics{
```

Comment:
I'm a little skeptical we'll actually look at / act on the per-round gauges, and timeseries are not free, but as long as it is just a few i guess it isn't too bad.